Mathlib/Data/Option/Basic.lean

```diff
@@ -280,8 +279,7 @@ theorem choice_eq_none (α : Type*) [IsEmpty α] : choice α = none :=
 
 end
 
--- Porting note: Can't simp tag this anymore because `elim` simplifies
--- @[simp]
+@[simp]
```

Comment:
Since this has a lambda at its head, the keys are just `.other` and this won't be useful.